src/ui-components/Buttons/styled.js

```diff
@@ -0,0 +1,28 @@
+import styled from 'styled-components';
+import {
+  View as NativeView,
+  TouchableOpacity as NativeTouchableOpacity,
+  Text as NativeText,
+  Image as NativeImage,
+} from 'react-native';
+
+export const TouchableOpacity = styled(NativeTouchableOpacity)`
+  align-items: center;
+`;
+
+export const View = styled(NativeView)`
+  flex-direction: row;
```

Comment:
All CSS properties should be alphabetically ordered. Do not do this manually, instead use some plugin (Lines Sorter etc) of your IDE